src/main.js

```diff
@@ -24,94 +29,107 @@ var currentCover;
 // Add your event listeners here 👇
 window.addEventListener("load", randomizePoster);
 randomCoverButton.addEventListener("click", randomizePoster);
-makeNewCoverButton.addEventListener("click", formViewPageHandler);
-savedCoversButton.addEventListener("click", savedCoversViewHandler);
 homeButton.addEventListener("click", homeViewPageHandler);
+makeNewCoverButton.addEventListener("click", formViewPageHandler);
+viewSavedCoversButton.addEventListener("click", viewSavedCoversHandler);
+makeBookButton.addEventListener("click", createNewBook);
 
 // Create your event handlers and other functions here 👇
 function getRandomIndex(array) {
   return Math.floor(Math.random() * array.length);
 }
 
 function randomizePoster() {
-  cover.src = covers[getRandomIndex(covers)],
-  title.innerText = titles[getRandomIndex(titles)],
-  firstDescriptor.innerText = descriptors[getRandomIndex(descriptors)],
+  cover.src = covers[getRandomIndex(covers)];
+  title.innerText = titles[getRandomIndex(titles)];
+  firstDescriptor.innerText = descriptors[getRandomIndex(descriptors)];
   secondDescriptor.innerText = descriptors[getRandomIndex(descriptors)]
 }
-
-function homeViewPageHandler(){
-  showHomeViewPage()
-  hideFormViewPage()
-  //hide home button:
-  toggleHomeButton()
-  //show new random cover button:
-  showRandomCoverButton()
-  //show save cover button:
-  toggleSaveCoverButton()
+function saveUserInput(){
+  covers.unshift(userCover.value);
+  titles.unshift(userTitle.value);
+  descriptors.unshift(userDescriptorOne.value);
+  descriptors.unshift(userDescriptorTwo.value);
 }
-
-function formViewPageHandler() {
-  hideHomeViewPage()
-  showFormViewPage()
-  hideRandomCoverButton()
-  hideSaveCoverButton()
-  showHomeButton()
+function createNewBook(){
+  event.preventDefault();
+  saveUserInput();
+  cover.src = covers[0];
+  title.innerText = titles[0];
+  firstDescriptor.innerText = descriptors[1];
+  secondDescriptor.innerText = descriptors[0];
+  homeViewPageHandler();
```

Comment:
We should create a function that will clear the user input data from the input fields after it has been stored & presented.